VideoWeb/VideoWeb.EventHub/Handlers/NewConferenceAddedEventHandler.cs

```diff
@@ -22,8 +23,10 @@ private async Task PublishNewConferenceAddedMessage(Guid conferenceId)
         {
             foreach (var participant in SourceConference.Participants)
```

Comment:
you can reduce the nesting if you do. Something Sonar tends to have a whinge about if there's to many layers
SourceConference.Participants.Where(x=>x.Role != StaffMember)